.github/workflows/build-docs.yaml

```diff
@@ -0,0 +1,51 @@
+name: Build docs
+
+on:
+  # run on push in main or rel_* branches excluding changes are only on example folders
+  push:
+    branches:
+      - main
+      - "rel_*"
+      # branches used to test the workflow
+      - "workflow_test_*"
+    paths-ignore:
+      - "examples/**"
```

Comment:
probably need to remove this, since examples are in the docs: https://docs.sqlalchemy.org/en/20/orm/examples.html